src/validator/mod.rs

```diff
@@ -0,0 +1,106 @@
+// Validator
+
+#![cfg(feature = "unstable-validator")]
+
+//! This module provides a DNSSEC validator as described in RFCs
+//! [4033](https://www.rfc-editor.org/info/rfc4033),
```

Comment:
I don't you mean tools.ietf.org. That doesn't work. Maybe you mean the datatracker? In any case I use what is used in the bibtex files. Those links seem the best for stable references.

Reply:
I don’t feel too strongly about which one we use, but we should be consistent.

But in any case, https://www.rfc-editor.org/info/rfc4033 is kind of terrible since you need an extra click to get to the text. If we use the RFC Editor site, it should probably be https://www.rfc-editor.org/rfc/rfc4033.html (i.e., rfc instead of info).

Reply:
This is the guide for references in RFCs: https://authors.ietf.org/en/references-in-rfcxml